the-collab-lab/tcl-30-whats-near-me
# [TCL30-66] Add a header in views

## Description

As user
I want to see a header in our views
so that I can navigate or search some locations by name or address

## Related Issue

[TCL30-66](https://the-collab-lab.atlassian.net/browse/TCL30-66)

## Acceptance Criteria

- Add a header bar at the top of the app.
- Navigate between views

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/8689897/131955189-311c12e1-26a5-4737-b486-18a9dbaeb6b3.png)

### After

![image](https://user-images.githubusercontent.com/8689897/131955145-f1ee4d70-126f-4b16-b83b-b82faea5fdbd.png)


## Testing Steps / QA Criteria

1. Check out `ac-implement-search-header` branch.
2. Navigate between views `map`, `list`, `settings`, and `help`
3. Search a place by name or street name.
